openassessment/xblock/static/js/src/studio/oa_edit_listeners.js

```diff
@@ -201,7 +201,7 @@ OpenAssessment.StudentTrainingListener.prototype = {
         if (changed) {
             this.displayAlertMsg(
                 gettext("Criterion Deleted"),
-                gettext("You've deleted a criterion. The system has removed the criterion from the sample responses in the Student Training step.")
+                gettext("You've deleted a criterion. The system has removed the criterion from the sample responses in the Learner Training step.")
```

Comment:
"You've deleted" -> "You have deleted"
"The system has removed..." -> "The criterion has been removed from the example responses..."
(note change from "sample responses" to "example responses".)